### PR TITLE
Added "JavaFindHelp" Modal

### PR DESCRIPTION
**Pull request by Eskaan**
- Java.js added link tag to Modal
- Java.js tweaked hight for link to fit in
- actions.js added reopenModal() function
- ChangeLogs.js added export to Container to use in Modal
- Modalsmanager.js added JavaFindHelp as new Modal
- Added JavaFindHelp.js - the new Modal

## Purpose
This Modal is made to help you find your local java executable, instead of  searching in the internet how to find it. 

## Features
- The Modal is opened by clicking on the new link under the java setting
- The Modal opens right away whit help your operating system
- Every operating system has its own help tab.

## How it looks
Modal whit MacOS

### CODE REVIEW

File: src/common/modals/ChangeLogs.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ const ChangeLogs = () => {
 
 export default memo(ChangeLogs);
 
-const Container = styled.div`
+export const Container = styled.div`
```
Comment (blarfoon) [inline src/common/modals/ChangeLogs.js]: Don't export components between components, just copy it over or extract it to a common file

File: src/common/modals/JavaFindHelp.js [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+const JavaFindHelp = (props = {}) => {
+  const platform = props.platform ? props.platform : process.platform;
+  const translateOS = os => {
+    const table = [
+      { platform: 'win32', name: 'Windows' },
+      { platform: 'darwin', name: 'MacOS' },
+      { platform: 'linux', name: 'Linux' }
+    ];
+    const found = table.find(item => item.platform === os);
```
Comment (blarfoon) [inline src/common/modals/JavaFindHelp.js]: you can use a hashtable here instead of an array

File: src/common/modals/JavaFindHelp.js [hunk 3]
```diff
@@ -0,0 +1,233 @@
+/* eslint-disable react/no-unescaped-entities */
+import React, { memo } from 'react';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components';
+import { Container } from './ChangeLogs';
+import Modal from '../components/Modal';
+import { reopenModal } from '../reducers/modals/actions';
+
+const JavaFindHelp = (props = {}) => {
```
Comment (blarfoon) [inline src/common/modals/JavaFindHelp.js]: Is this prop even passed?

File: src/common/reducers/modals/actions.js [hunk 4]
```diff
@@ -39,3 +39,21 @@ export function closeAllModals() {
     }, 220);
   };
 }
+
+export function reopenModal(modalType, modalProps = {}) {
```
Comment (blarfoon) [inline src/common/reducers/modals/actions.js:43]: What's this for?

**Review by blarfoon — CHANGES_REQUESTED** [review]

File: src/common/reducers/modals/actions.js (hunk 4 above)
Comment (Eskaan) [reply src/common/reducers/modals/actions.js:43]: Uhm… could be a leftover from an Experiment 😅

Comment (Eskaan) [reply src/common/reducers/modals/actions.js:43]: Ah no, that is a function who reopens an already opened modal whit new props. That’s made to get an animation when switching the os.

**Comment by blarfoon** [comment]
Does this have instructions for mac/linux/windows? Can you fix conflicts?

**Comment by Eskaan** [comment]
Do we (still) need this feature?

**Comment by Eskaan** [comment]
No longer needed

**Pull request closed without merging**